src/services/db/GitFileSystemService.ts

```diff
@@ -539,6 +539,29 @@ export default class GitFileSystemService {
         )
         .orElse(() =>
           // Retry push once
+          ResultAsync.fromPromise(
```

Comment:
@harishv7 ps should have been clearer, specificly here, why do we have a retry here ah? + why is it on the second retry we pass in diferent args?

Reply:
Oh this is just cos in case the first attempt fails, we have a backup to retry once more.

regarding the options, I think there may be a bug - we should be passing in the same options both for first & 2nd attempts

Reply:
do you know why it (i think consistently) fails on file rename?

Reply:
> do you know why it (i think consistently) fails on file rename?

hmm, consistent failure on 1st try is not an expected behaviour